texts/formatted xkcd.txt

```diff
@@ -0,0 +1,2112 @@
+"<xkcd>"
+	"xkcd" np al sg 
+
+"<,>"
+	"," cm  
+ 
+"<sometimes>"
+	"sometimes" adv  
+ 
+"<styled>"
+	"style" vblex past  
+;	"style" vblex pp  
+ 
+"<XKCD>" 
+	"XKCD" np al sg 
+
+"<,>"
+	"," cm  
+ 
+"<is>"
+	"be" vbser pres p3 sg  
+ 
+"<a>"
+	"a" det ind sg  
+ 
+"<webcomic>"
+	"webcomic" n sg 
+ 
+"<created>"
+;	"created" adj  
+;	"create" vblex past  
+	"create" vblex pp  
+ 
+"<by>"
+;	"by" adv  
+	"by" pr  
+ 
+"<American>"
+	"American" adj 
+;	"American" n sg  
+;	"American" n sg  
+ 
+"<author>"
+	"author" n sg  
+;	"author" vblex inf  
+;	"author" vblex pres  
+;	"author" vblex imp  
+ 
+"<Randall>"
+;	"Randall" np cog sg  
+	"Randall" np ant m sg  
+ 
+"<Munroe>"
+	"Munroe" np cog sg 
+
+"<.>"
+	"." sent  
+ 
+"<The>"
+	"The" det def sp  
+ 
+"<comic>"
+	"comic" n sg  
+
+"<'s>"
+	"'s" gen  
+;	"be" vbser pres p3 sg  
+;	"have" vbhaver pres p3 sg  
+ 
+"<tagline>"
+	"tagline" n sg  
+ 
+"<describes>"
+	"describe" vblex pres p3 sg  
+ 
+"<it>"
+;	prpers prn subj p3 nt sg  
+	prpers prn obj p3 nt sg  
+ 
+"<as A>" 
+	"as a" pr  

```

Comment:
Is this right?

Reply:
I guess the words didn't get split. I'll manually separate them, I guess